modules/AOR_Reports/AOR_Report_Before.js

```diff
@@ -172,7 +172,8 @@ $(document).ready(function(){
           if(relData[field]['type'] == 'relationship') {
             modulePath = field;
             if (node) {
-              modulePath = node.module_path + ":" + field;
+              if (node.module_path)
```

Comment:
Please use if statements with starting and ending curly braces. 

For-example:

```
if (node.module_path) {
    modulePath = node.module_path + ":" + field;
}
```

Reply:
thanks, done.